### PR TITLE
SSCS-10295 change interpreter language type to dynamic list

### PR DESCRIPTION
**Pull request by pk2453**
JIRA link (if applicable)
https://tools.hmcts.net/jira/browse/SSCS-10295

Change description
Interpreter requirements set in the adjournment notice is included in new hearing request

Does this PR introduce a breaking change? (check one with "x")

[x] Yes
[ ] No

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/AdjournCaseDaysOffset.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum AdjournCaseDaysOffset {
+    OTHER(0, "Other"),
```
Comment (nickhill111) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/AdjournCaseDaysOffset.java:10]: OTHER should go at the bottom

**Review by nickhill111 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/service/VenueService.java [hunk 2]
```diff
@@ -24,13 +24,17 @@ public String getEpimsIdForVenue(String processingVenue) {
...
+
+    public String getEpimsIdForVenueId(String venueId) {
         VenueDetails venueDetails = venueDataLoader.getVenueDetailsMap()
             .get(String.valueOf(venueId));
 
         return Optional.ofNullable(venueDetails)
             .map(VenueDetails::getEpimsId)
             .orElseThrow(() -> {
-                throw new IllegalStateException("Unable to find epims ID for processing venue: " + processingVenue);
+                throw new IllegalStateException("Unable to find epims ID for venue ID: " + venueId);
```
Comment (amoslv1) [inline src/main/java/uk/gov/hmcts/reform/sscs/service/VenueService.java:37]: These changes have been reverted in the base branch 11092. This was done to keep features separate, as these belong to @nickhill111 ticket 10293.

File: src/test/java/uk/gov/hmcts/reform/sscs/ccd/domain/AdjournCaseDaysOffsetTest.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+class AdjournCaseDaysOffsetTest {
```
Comment (amoslv1) [inline src/test/java/uk/gov/hmcts/reform/sscs/ccd/domain/AdjournCaseDaysOffsetTest.java:8]: All these tests have been deleted. Please rebase.

**Review by amoslv1 — CHANGES_REQUESTED** [review]
Please rebase, all changes in this PR which aren't specifically yours should not be here

File: src/test/java/uk/gov/hmcts/reform/sscs/ccd/domain/AdjournCaseDaysOffsetTest.java (hunk 3 above)
Comment (pk2453) [reply src/test/java/uk/gov/hmcts/reform/sscs/ccd/domain/AdjournCaseDaysOffsetTest.java]: rebased and only relevant changes now.

**Review by amoslv1 — APPROVED** [review]

**Pull request merged**